### PR TITLE
remove assumptions about home currency and currency scales

### PR DESCRIPTION
**Pull request by scionaltera**

### CODE REVIEW

File: src/main/java/com/r307/arbitrader/service/TradingService.java [hunk 1]
```diff
@@ -914,11 +915,15 @@ BigDecimal getMaximumExposure(Exchange ... exchanges) {
     }
 
     // log the balances of two exchanges and the sum of both
-    private BigDecimal logCurrentExchangeBalances(Exchange longExchange, Exchange shortExchange) {
+    private BigDecimal logCurrentExchangeBalances(final Exchange longExchange, final Exchange shortExchange) {
```
Comment (dazito) [inline src/main/java/com/r307/arbitrader/service/TradingService.java:918]: We should change this method name because it does not only logs but also get and returns total balance. But maybe we should do this in a separate PR

**Review by dazito — APPROVED** [review]

**Review by AllirionX — APPROVED** [review]
